### PR TITLE
[Terraform] Editing ec_deployment_traffic_filter.md

### PR DESCRIPTION
**Pull request by alaudazzi**
## Description
This PR proposes some editing suggestions.

## Related Issues

#78

## Motivation and Context
The output was tested with https://registry.terraform.io/tools/doc-preview and looks OK.
The text was reviewed against the [Elastic writing style guide](https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide) and [Terraform guidelines](https://www.terraform.io/docs/registry/providers/docs.html).

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improves code quality but has no user-facing effect)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

### CODE REVIEW

File: docs/resources/ec_deployment_traffic_filter.md [hunk 1]
```diff
@@ -54,33 +54,31 @@ resource "ec_deployment_traffic_filter" "example" {
...
+* `type` - (Required) Type of the ruleset.  It can be `"ip"` or `"vpce"`.
+* `region` - (Required) Filter region, the ruleset can only be attached to deployments in the specific region.
+* `rule` (Required) Rule block, which can be specified multiple times for multiple rules.
+* `include_by_default` - (Optional) To automatically include the ruleset in the new deployments. Defaults to `false`.
+* `description` - (Optional) Description of the ruleset.
 
 ### Rules
 
-The `rule` supports the following:
+The `rule` supports the following configuration options:
```
Comment (karencfv) [inline docs/resources/ec_deployment_traffic_filter.md:66]:
```suggestion
The `rule` block supports the following configuration options:
```

File: docs/resources/ec_deployment_traffic_filter.md [hunk 2]
```diff
@@ -54,33 +54,31 @@ resource "ec_deployment_traffic_filter" "example" {
...
 
-* `source` - (Required) source (IP or VPC endpoint) which the ruleset will accept traffic from.
-* `description` - (Optional) description to attach to this individual rule.
+* `source` - (Required) Source type, IP or VPC endpoint, from which the ruleset accepts traffic.
```
Comment (karencfv) [inline docs/resources/ec_deployment_traffic_filter.md:68]:
```suggestion
* `source` - (Required) Source type, `"ip"` or `"vpce"`, from which the ruleset accepts traffic.
```

**Review by marclop — APPROVED** [review]
Looks good

**Review by karencfv — APPROVED** [review]
🙇‍♀️

**Comment by alaudazzi** [comment]
@elasticmachine update branch

**Pull request merged**